rllib/algorithms/algorithm_config.py

```diff
@@ -2437,6 +2441,9 @@ def offline_data(
                 schema used is `ray.rllib.offline.offline_data.SCHEMA`. If your data set
                 contains already the names in this schema, no `input_read_schema` is
                 needed.
+            intput_compress_columns: What input columns are compressed with LZ4 in the
```

Comment:
typo: "intput_" -> "input_"

Reply:
Great catch! I got this error multiple times, but in code I got exceptions :)